### ISSUE TITLE
SA1111 (ClosingParenthesisMustBeOnLineOfLastParameter) triggers on brackets used for attributes

### ISSUE THREAD

**Issue by vweijsters** [DotNetAnalyzers/StyleCopAnalyzers#808]
The following code should not but does trigger SA1111:

```C#
    [
    Export
    ]
    public class Foo
    {
    }
```

Stylecop does not trigger for this, inline with the documentation.
